docs/ray_cluster_interaction.md

```diff
@@ -0,0 +1,71 @@
+# Ray Cluster Interaction
+
+The CodeFlare SDK offers multiple ways to interact with Ray Clusters including the below methods.
+
+## get_cluster()
+The `get_cluster()` command is used to initialise a `Cluster` object from a pre-existing Ray Cluster/AppWrapper. <br>
+Below is an example of it's usage:
+```
+from codeflare_sdk import get_cluster
+cluster = get_cluster(cluster_name="raytest", namespace="example", is_appwrapper=False, write_to_file=False)
+-> output: Yaml resources loaded for raytest
+
+cluster.status()
+-> output:
+                    🚀 CodeFlare Cluster Status 🚀
+
+ ╭─────────────────────────────────────────────────────────────────╮
+ │   Name                                                          │
+ │   raytest                                           Active ✅   │
+ │                                                                 │
+ │   URI: ray://raytest-head-svc.example.svc:10001                 │
+ │                                                                 │
+ │   Dashboard🔗                                                   │
+ │                                                                 │
+ ╰─────────────────────────────────────────────────────────────────╯
+(<CodeFlareClusterStatus.READY: 1>, True)
+
+cluster.down()
+
+cluster.up() # This method will create an exact copy of the retrieved Ray Cluster only if the Ray Cluster has been previously deleted.
+```
+
+These are the parameters the `get_cluster()` method accepts:
+* `cluster_name: str # Required` -> The name of the Ray Cluster.
+* `namespace: str # Default: "default"` -> The namespace the Cluster.
+* `is_appwrapper: bool # Default: False` -> The function will attempt to retrieve an AppWrapper instead of a Ray Cluster.
```

Comment:
```suggestion
* `is_appwrapper: bool # Default: False` -> When set to `True` the function will attempt to retrieve an AppWrapper instead of a Ray Cluster.
```